src/server/pegasus_server_impl.cpp

```diff
@@ -1865,6 +1961,16 @@ ::dsn::error_code pegasus_server_impl::start(int argc, char **argv)
     _cu_calculator = dsn::make_unique<capacity_unit_calculator>(this);
     _server_write = dsn::make_unique<pegasus_server_write>(this, _verbose_log);
 
+    ::dsn::tasking::enqueue_timer(LPC_ANALYZE_HOTKEY,
+                                  &_tracker,
+                                  [this]() { _read_hotkey_collector->analyse_data(); },
```

Comment:
`_read_hotkey_collector` is a private variable, I can't put it into lambda directly